journey-map/README.md

```diff
@@ -28,6 +28,15 @@ Use `npm start` to run the built code.
 
 In production, the journey map uses `../journey-map.Dockerfile` to run these steps.
 
+### Visualising user traffic
+
+Run the map locally, but set the following environment variables, for the deployment of your choice! Check API Gateway for the endpoint and API key info.
+```bash
+export JOURNEY_TRANSITIONS_ENDPOINT=...
+export ANALYTICS_API_KEY=...
+npm run dev
+```
+
```

Comment:
Could we add a bit more commentary here referencing the Analytics APIGW and new lambda?